packages/devtools_app/lib/src/shared/feature_flags.dart

```diff
@@ -62,7 +62,7 @@ abstract class FeatureFlags {
   /// DevTools screen and the standalone tool for IDE embedding.
   ///
   /// https://github.com/flutter/devtools/issues/6013
-  static bool deepLinkValidation = enableExperiments;
```

Comment:
Is there a reason the `enableExperiements` value is not guarded enough? If it is false, the only way to test this feature will be to manually flip the flag, vs using the existing devtools + experiments run config in VS code. If you don't want to run DevTools with experiments enabled, you can run using the devtools run config that doesn't have "+ experiments"